community/frag.md

```diff
@@ -1,6 +1,7 @@
 ---
 tag: frag
-alias: ["jemand-da", "fragnicht"]
+alias: ["frag-einfach", "frageinfach", "jemand-da", "fragnicht"]
 ---
 
+https://nohello.net
```

Comment:
Would personally put the link at the bottom for aesthetic reasons (the text is more important and concise than the website), but either way works